src/SFA.DAS.EAS.Portal/DependencyResolution/ApplicationServices.cs

```diff
@@ -1,13 +1,38 @@
-﻿using Microsoft.Extensions.DependencyInjection;
+﻿using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using SFA.DAS.CommitmentsV2.Messages.Events;
+using SFA.DAS.EAS.Portal.Application.Adapters;
 using SFA.DAS.EAS.Portal.Application.Commands;
+using SFA.DAS.EAS.Portal.Application.Commands.Cohort;
+using SFA.DAS.EAS.Portal.Application.Commands.Reservation;
+using SFA.DAS.EAS.Portal.Application.Services;
 
 namespace SFA.DAS.EAS.Portal.DependencyResolution
 {
     public static class ApplicationServices
     {
         public static IServiceCollection AddApplicationServices(this IServiceCollection services)
         {
-            return services.AddTransient<AddReservationCommand>();
+            services.AddTransient<AddReservationCommand>();
+            var configuration = services.BuildServiceProvider().GetService<IConfiguration>();
+
+            services.AddCommitmentsApiConfiguration(configuration);
```

Comment:
I don't think this belongs as part of application services, I think `AddCommitmentsApiConfiguration` should be added separately when creating the host in `Program.cs`.